charts/pulsar/templates/tls/tls-certs-internal.yaml

```diff
@@ -66,6 +66,44 @@ spec:
 {{- end }}
 {{- end }}
 
+{{- if or .Values.tls.zookeeper.enabled .Values.tls.bookie.enabled }}
+apiVersion: cert-manager.io/v1alpha2
```

Comment:
Has this API version been deprecated in recent cert-manager versions?

Reply:
that's right, but seems all cert in this template is in v1alpha2 version, will have a separate pr to upgrade to v1 version.